resources/views/partials/navigation.blade.php

```diff
@@ -48,12 +48,6 @@ class="dropdown-toggle"
                 <ul class="dropdown-menu" role="menu">
 
                     @auth
-                        <li>
```

Comment:
i think its a bad idea to remove the link in the navigation
i think it would be better to provide both ways to access the personal page
some people just click on their names and some searching for a menu entry

Reply:
I kinda want to put everything into the personal page, incl language switcher, password change, etc.

Reply:
This would make the gear menu obsolete for non-management users

Reply:
And additionally move the lara update from the "diamond-menu" to "gear-menu" for admins

Reply:
I agree with @4D44H . Furthermore, users are not trained to use the gear menu, as currently the only use of this menu for normal users is for changing languages. We can probably assume that users don't change languages that often. 
But I also see your concern about offering multiple ways to get to the user page. What about adding an icon to the navbar (not inside the cog menu, but as a top level button) which uses something like this icon: https://fontawesome.com/icons/user?style=solid ?